consensus/src/epoch_manager.rs

```diff
@@ -759,8 +761,9 @@ impl EpochManager {
             self.time_service.clone(),
             genesis_block_id,
         );
-
-        tokio::spawn(dag_driver.start());
+        let (close_tx, close_rx) = oneshot::channel();
+        self.dag_driver_close_tx = Some(close_tx);
+        tokio::spawn(dag_driver.start(close_rx));
```

Comment:
We should call it from shutdown_current_processor

Reply:
Fixed.